tests/testsuite/crystal.rs

```diff
@@ -0,0 +1,54 @@
+use ansi_term::Color;
+use std::fs::{self, File};
+use std::io;
+
+use crate::common;
+
+#[test]
```

Comment:
While we're going to be replacing the test framework with the new mocking framework soon, I'm fine with leaving these in place for now so that we don't have to come up with them again when we do the move.